test/src/kbaserelationengine/main/test/MainObjectProcessorTest.java

```diff
@@ -23,56 +32,90 @@
-        String tokenStr = props.getProperty("secure.indexer_token");
-        AuthToken kbaseIndexerToken = authSrv.validateToken(tokenStr);
-        String mongoHost = props.getProperty("secure.mongo_host");
-        int mongoPort = Integer.parseInt(props.getProperty("secure.mongo_port"));
-        String elasticHost = props.getProperty("secure.elastic_host");
-        int elasticPort = Integer.parseInt(props.getProperty("secure.elastic_port"));
-        String esUser = props.getProperty("secure.elastic_user");
-        String esPassword = props.getProperty("secure.elastic_password");
-        HttpHost esHostPort = new HttpHost(elasticHost, elasticPort);
-        if (cleanup) {
-            deleteAllTestMongoDBs(mongoHost, mongoPort);
-            deleteAllTestElasticIndices(esHostPort, esUser, esPassword);
-        }
-        String kbaseEndpoint = props.getProperty("kbase_endpoint");
-        URL wsUrl = new URL(kbaseEndpoint + "/ws");
-        File typesDir = new File("resources/types");
-        File tempDir = new File("test_local/temp_files");
-        if (!tempDir.exists()) {
-            tempDir.mkdirs();
+        TestCommon.stfuLoggers();
+        final URL authURL = TestCommon.getAuthUrl();
+        final URL authServiceRootURL;
+        if (authURL.toString().contains("api")) {
+            authServiceRootURL = new URL(authURL.toString().split("api")[0]);
+            System.out.println(String.format("Using %s as auth root URL", authServiceRootURL));
+        } else {
+            authServiceRootURL = authURL;
         }
-        String mongoDbName = "test_" + System.currentTimeMillis() + "_DataStatus";
-        String esIndexPrefix = "test_" + System.currentTimeMillis() + ".";
-        mop = new MainObjectProcessor(wsUrl, kbaseIndexerToken, mongoHost,
-                mongoPort, mongoDbName, esHostPort, esUser, esPassword, esIndexPrefix, 
-                typesDir, tempDir, false, new LineLogger() {
+        final ConfigurableAuthService authSrv = new ConfigurableAuthService(
+                new AuthConfig().withKBaseAuthServerURL(authURL));
+        final AuthToken kbaseIndexerToken = TestCommon.getToken(authSrv);
+
+        tempDir = Paths.get(TestCommon.getTempDir()).resolve("MainObjectProcessorTest");
+        // should refactor to just use NIO at some point
+        FileUtils.deleteQuietly(tempDir.toFile());
+        tempDir.toFile().mkdirs();
+
+        // set up mongo
+        mongo = new MongoController(
+                TestCommon.getMongoExe(),
+                tempDir,
+                TestCommon.useWiredTigerEngine());
+        mc = new MongoClient("localhost:" + mongo.getServerPort());
+        final String dbName = "DataStatus";
+        db = mc.getDatabase(dbName);
+        
+        // set up elastic search
+        es = new ElasticSearchController(TestCommon.getElasticSearchExe(), tempDir);
+        
+        // set up Workspace
+        ws = new WorkspaceController(
+                "0.7.2-dev1",
```

Comment:
Sure, I can do that, although this is currently the only version of the workspace that'll work with RESKE once it's updated to use the admin commands. I'll do it in the next PR if that's ok with you.